<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YearCal</class>
 <widget class="QDialog" name="YearCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="0" colspan="6">
    <widget class="QPushButton" name="checkEvent">
     <property name="text">
      <string>Узнать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Введите год:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox3">
     <property name="text">
      <string>3 x 4</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Введите месяц: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="checkBox6">
     <property name="text">
      <string> 6 x 2</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="day"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Введите день:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Формат календаря: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="year"/>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Узнать событие дня</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLineEdit" name="month"/>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="checkBox4">
     <property name="text">
      <string> 4 x 3</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="6">
    <widget class="QPushButton" name="back">
     <property name="text">
      <string>Вернуться на главный экран</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="calOut">
     <property name="text">
      <string>Вывести календарь</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Выделить особые даты</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
